yarn.lock

```diff
@@ -1368,6 +1368,20 @@
     prop-types "^15.7.2"
     qs "^6.9.1"
 
+"@opentripplanner/core-utils@^1.2.1":
```

Comment:
Please use [this tool](https://www.npmjs.com/package/yarn-deduplicate) to fix deduplicate this entry in yarn.lock.